marvin.py

```diff
@@ -109,15 +111,25 @@ def is_message_in_correct_group(self, chat: Chat):
         """
         return chat.id == self.authorized_group_id
 
-    def add_default_comment(self, post_submission):
+    def add_default_comment(self, post_submission, tg_msg_id):
```

Comment:
Non la chiami mai dandogli il terzo parametro mi sembra, hai cambiato il corpo ma non le chiamate

Reply:
Si invece, guarda la riga 229 vecchia 
`self.add_default_comment(submission)`
che è diventata 242
`self.add_default_comment(submission, update.message.message_id)`
ad esempio

Reply:
Hai ragione, mi ero confuso un momento, ora però ho altre domande:

- Lo hai testato in qualche modo o devo testarlo io (?)

- `update.message.message_id` (che tra l'altro è come ho fatto pure io in comment) mi ha messo un dubbio, quel messaggio non è quello con il comando che verrà eliminato?

Reply:
Ho testato tutto e funziona, però non ho provato, hai ragione tu, a cacciare nel mio gruppo di test anche il bot cancellatore di comandi. Come dici tu, il commento si perderebbe. Tuttavia non deve perdersi, quindi il cancellatore (che sarà implementato in #15) non cancellerà i commenti, quindi su questo punto siamo a posto 👍

Reply:
Quindi il cancellatore non cancellerà i comandi di marvin? Allora ok, comunque per ora mi sembra a posto, domani do una riletta e la accetto

Reply:
Il cancellatore è marvin stesso, in futuro. Sicuramente i comandi comment non saranno cancellati, a meno che non validi

Reply:
Il bot che abbiamo ora nel gruppo è un bot temporaneo, che cancella tutto, ma si può disattivare in qualsiasi momento, non centra con #15